### ISSUE TITLE
[Obsidian] Error: Worker terminated due to reaching memory limit

### ISSUE THREAD

**Issue by Garvnanwani** [raycast/extensions#10032]
### Extension

https://www.raycast.com/KevinBatdorf/obsidian

#### Error:
```
Error: Worker terminated due to reaching memory limit: JS heap out of memory

NodeError:errors:406:5
[kOnExit]:worker:313:26
Worker.<computed>.onexit:worker:229:20
```

### Steps To Reproduce

My Obsidian has more than 5000 notes, none of the extension commands works.
I am on latest version of the extension

**Comment by pernielsentikaer** [comment]
@KevinBatdorf could we make use of the new pagination we shipped in the API in 1.69.0?

**Comment by KevinBatdorf** [comment]
I'll look into it this week. Thanks for point that out
